sdk/monitor/azquery/CHANGELOG.md

```diff
@@ -3,12 +3,15 @@
 ## 1.2.0-beta.2 (Unreleased)
 
 ### Features Added
+* Added `AutoAdjustTimegrain`, `RollUpBy`, and `ValidateDimensions` options to `MetricsClient.QueryResource`
 
 ### Breaking Changes
+* Removed `MetricsBatchClient`
```

Comment:
We should point them to azmetrics for this.

Reply:
added in [8b6ce86](https://github.com/Azure/azure-sdk-for-go/pull/24476/commits/8b6ce862d9a6cea8250e28edb511d8e9db7902fd)